test/TensorValuesTests/IndexingTests.jl

```diff
@@ -14,6 +14,7 @@ v = VectorValue{4}(a)
 @test size(v) == (4,)
 @test length(v) == 4
 @test lastindex(v) == length(v)
+@test v[end] == a[end]
```

Comment:
where do these tests come from? shouldnt be these already in `master` ? (i.e., is your branch up to date with `master`?) they seem to be unrelated to this PR.

Reply:
No, I forgot adding these tests, sorry took the opportunity to add it now (instead of again creating a PR), sorry for the confusion, it is not under PR.

Reply:
ok, no worries ... we may merge them here, but the cleaner way to go is to do it in a separate PR

Reply:
Yes I am sorry, will be careful next time and separate it into different PR. Probably would be too messy here now, to revert back?

Reply:
Better not to revert back, I agree. 

For completeness, perhaps we can add in the NEWS.md a comment that we added some missing tests missing from another PR.

Reply:
Definitely, will mention it in the `NEWS.md`